### PR TITLE
Reverted python id

### PR DESCRIPTION
**Pull request by schultzp2020**
Signed-off-by: Paul Schultz <pschultz@pobox.com>

### What does this PR do?:
Reverts python id

### Which issue(s) this PR fixes:
https://github.com/devfile/registry/pull/123 changed the python id and broke parent support. 

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

**Comment by elsony** [comment]
You may also want to test out on the ODC case to make sure the changes fixes the problem on ODC sample scenario as well.

**Comment by maysunfaisal** [comment]
/hold

until ODC test confirmed

**Comment by maysunfaisal** [comment]
Cannot test on ODC because this is a stack. 
If one needs to test on ODC, then you need to update sample

File: stacks/python/devfile.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 schemaVersion: 2.1.0
 metadata:
-  name: python-flask
+  name: python
   version: 2.0.1
```
Comment (kadel) [inline stacks/python/devfile.yaml:4]: Shouldn't the version be incremented in big changes like this?

**Pull request merged**